### ISSUE TITLE
fluvio cluster startup instability in CD

### ISSUE THREAD

**Issue by tjtelan** [infinyon/fluvio#1480]
In the `CD_Dev` workflow, the cross platform test fails due to a problem with the cluster startup.

```
Starting cluster
✅ ok: Kubernetes config is loadable
✅ ok: Supported helm version is installed
✅ ok: Fixed: Missing Fluvio system charts.
✅ ok: Previous fluvio installation not found
installing fluvio chart
Fluvio SC is up at: 172.18.0.2:32031
Error: 
   0: Fluvio cluster error
   1: Failed to install Fluvio on Kubernetes
   2: Fluvio client error
   3: Fluvio socket error
   4: Connection refused (os error 111)
```

**Comment by sehz** [comment]
caused by #1453

**Comment by sehz** [comment]
Upgrade test will be fixed in 0.9.6 since installer's instability is fixed in 0.9.5

**Comment by sehz** [comment]
fixed
